src/patternfly/components/Popover/popover.scss

```diff
@@ -94,11 +129,78 @@
       transform: translateX(var(--pf-c-popover__arrow--m-right--TranslateX)) translateY(var(--pf-c-popover__arrow--m-right--TranslateY)) rotate(var(--pf-c-popover__arrow--m-right--Rotate));
     }
   }
+
+  &.pf-m-icon {
+    display: flex;
+  }
+
+  &.pf-m-default {
+    --pf-c-popover__title-icon--Color: var(--pf-c-popover--m-default__title-icon--Color);
+  }
+
+  &.pf-m-danger {
+    --pf-c-popover__title-icon--Color: var(--pf-c-popover--m-danger__title-icon--Color);
+  }
+
+  &.pf-m-warning {
+    --pf-c-popover__title-icon--Color: var(--pf-c-popover--m-warning__title-icon--Color);
+  }
+
+  &.pf-m-success {
+    --pf-c-popover__title-icon--Color: var(--pf-c-popover--m-success__title-icon--Color);
+  }
+
+  &.pf-m-info {
+    --pf-c-popover__title-icon--Color: var(--pf-c-popover--m-info__title-icon--Color);
+  }
+}
+
+.pf-c-popover__title,
+.pf-c-popover__title-text {
+  @include pf-text-overflow;
+}
+
+.pf-c-popover__title {
+  flex: 0 0 auto;
+  font-family: var(--pf-c-popover__title--FontFamily);
+  font-size: var(--pf-c-popover__title--FontSize);
+  line-height: var(--pf-c-popover__title--LineHeight);
+
+  &.pf-m-icon {
+    display: flex;
+  }
+
+  &.pf-m-default {
+    color: var(--pf-c-title--m-default__title--Color);
+  }
+
+  &.pf-m-danger {
+    color: var(--pf-c-title--m-danger__title--Color);
+  }
+
+  &.pf-m-warning {
+    color: var(--pf-c-title--m-warning__title--Color);
+  }
+
+  &.pf-m-success {
+    color: var(--pf-c-title--m-success__title--Color);
+  }
+
+  &.pf-m-info {
+    color: var(--pf-c-title--m-info__title--Color);
+  }
+}
+
+.pf-c-popover__title-icon {
+  margin-right: var(--pf-c-popover__title-icon--MarginRight);
+  color: var(--pf-c-popover__title-icon--Color);
 }
 
 .pf-c-popover__content {
   position: relative;
-  padding: var(--pf-c-popover__content--PaddingTop) var(--pf-c-popover__content--PaddingRight) var(--pf-c-popover__content--PaddingBottom) var(--pf-c-popover__content--PaddingLeft);
+  padding:
```

Comment:
Can you move these values so everything is on a single line?

Reply:
Fixed